### ISSUE TITLE
my question is nn.InstanceNorm2d(),the value of num_features

### ISSUE THREAD

**Issue by hottestme** [ChangeZH/Pytorch_Transformer_Models#1]
in MSA.py at 75
self.num_features = (self.input_size / (2 ** (self.stage + 2))) ** 2 * (
                self.input_size / (2 ** (self.stage + 3))) ** 2 if self.input_size / (
                2 ** (self.stage + 3)) % 4 == 0 else (self.input_size / (2 ** (self.stage + 2))) ** 2 * int(
            self.input_size / (2 ** (self.stage + 3)) + 1) ** 2

        self.Conv = nn.Sequential(nn.Conv2d(in_channels=2 ** self.stage, out_channels=2 ** self.stage,
                                            kernel_size=3, stride=1, padding=1),
                                  nn.Softmax(-1),
                                  nn.InstanceNorm2d(num_features=int(self.num_features)))
num_features got a very big number
but
i find most blogs，the value of num_features is feature's channel
but but
the code is running very ok
help help
